feat: settings to skip init and ephemeral containers. (#74)
kubewarden/pod-privileged-policy: README.md

# Settings

The policy has two configurations:

> - `skip_init_containers`: if set to `true` instructs the policy to ignore if

Reviewer: ```suggestion
- `skip_init_containers`: if set to `true` instructs the policy to ignore that
```

>   some init container is configure as privileged. Default value is `false`

Reviewer: ```suggestion
  some init container is configured as privileged. Default value is `false`
```

> - `skip_ephemeral_containers`:if set to `true` instructs the policy to ignore

Reviewer: ```suggestion
- `skip_ephemeral_containers`: if set to `true` instructs the policy to ignore
```

>   if some ephemeral container is configure as privileged. Default value is

Reviewer: ```suggestion
  that some ephemeral container is configured as privileged. Default value is
```
